pkg/registry/apis/provisioning/jobs/migrate/users.go

```diff
@@ -14,11 +14,14 @@ import (
 )
 
 func (j *migrationJob) loadUsers(ctx context.Context) error {
-	client := j.client.Resource(schema.GroupVersionResource{
+	client, _, err := j.parser.Clients().ForResource(schema.GroupVersionResource{
```

Comment:
now that we are touching this. Can we have a common place to create specific ones as well? I am talking about folders, dashboards, users, etc.